WrightTools/kit.py

```diff
@@ -637,7 +644,7 @@ def read_headers(filepath):
 
 
 def write_h5(filepath, dictionary):
-    """ Save a python dictionary into an `HDF5 <https://www.hdfgroup.org/HDF5/doc/H5.intro.html>`_
+    """ Save a python dictionary into an `HDF5`_
```

Comment:
One thing we could do is use anonymous hyperlink in this case, i.e. `__` and then have 
```
.. __: hdf5-write_
.. _hdf5-write: https://hdf5.org/path/to/index.html
```